recipes/fit-sne/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+export CPPFLAG="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
```

Comment:
These should inherit from the flag in the environment:

```
export CPPFLAG="$CXXFLAGS -I$PREFIX/include"
export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
```

Reply:
Okay, resolved in latest push.